llvm/lib/CodeGen/SelectionDAG/SelectionDAGDumper.cpp

```diff
@@ -104,6 +104,7 @@ std::string SDNode::getOperationName(const SelectionDAG *G) const {
   case ISD::ATOMIC_STORE:               return "AtomicStore";
   case ISD::PCMARKER:                   return "PCMarker";
   case ISD::READCYCLECOUNTER:           return "ReadCycleCounter";
+  case ISD::READSTEADYCOUNTER:             return "ReadFixedTimer";
```

Comment:
Should it be "ReadSteadyCounter" ? 

Also, whitespace/alignment looks off.

Reply:
Forgot to fix that when I renamed it, thanks for spotting it.